cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -137,7 +138,8 @@
   private final SchedulerQueueResolver schedulerQueueResolver;
 
   // Handles that are running, or not yet completely fetched, they have longer timeout
-  private final Cache<QueryHandle, OperationInfo> activeHandleCache;
+  @VisibleForTesting
+  public final Cache<QueryHandle, OperationInfo> activeHandleCache;
```

Comment:
Not a good idea to make a member accessible publicly even with an annotation. Instead can you run a query using explore service, and then verify active count?